esprit/raw.py

```diff
@@ -350,3 +358,25 @@ def refresh(connection):
     url = elasticsearch_url(connection, endpoint="_refresh")
     resp = _do_post(url, connection)
     return resp
+
+##############################################################
+## Aliases
+
+def to_alias_actions(add=None, remove=None):
+    """
+    Create a list of actions to post to the alias endpoint
+    :param add: a list of dicts: [{"alias": <alias name>, "index": <index name>}] to add to the alias list
+    :param remove: a list of dicts: [{"alias": <alias name>, "index": <index name>}] to remove from the alias list
+    :return: The actions packaged together, as {"actions": [...]}
+    """
+    acts = {"actions": []}
+    if add is not None:
```

Comment:
Might be slightly more idiomatic to write something like

```python
add = add if add else []
remove = remove if remove else []
[acts["actions"].append({"add": a}) for a in add]  # unchanged
[acts["actions"].append({"remove": r}) for r in remove]  # unchanged
```
and drop the `if add is not None` and `if remove is not None`.

But this is perfectly fine!